features/machine/machine_healthcheck.feature

```diff
@@ -166,5 +166,44 @@ Feature: MachineHealthCheck Test Scenarios
      | Ensuring a requeue happens in 15m0  |
     """
    Then the machine should be remediated
+ 
+  # @author miyadav@redhat.com
+  # @case_id OCP-29857
+  @admin
+  @destructive
+  Scenario: [MHC] MaxUnhealthy should not allow malformed values
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    Then I use the "openshift-machine-api" project
+    And admin ensures machine number is restored after scenario
+
+    Given I clone a machineset and name it "machineset-clone-29857"
+
+    # Create MHC with malformed unhealthy nodes value
+    When I run oc create over "<%= BushSlicer::HOME %>/testdata/cloud/mhc/mhc_malformed.yaml" replacing paths:
```

Comment:
Replace this line with
```
Given I obtain test data file "xxx"
When I run oc create over "xxx" replacing paths:
```

Reply:
since I am only changing namespace ,do i need to obtain file or this way is also ok ?

Reply:
I was suggested to use the above steps in a PR several days back, so I thought that had some advantage, I didn't look into their step definitions though. @liangxia Can you please suggest?

Reply:
@jhou1 @miyadav the even though it's a two steps process, there are two advantages

1. getting data files in abstracted way so that if in the future `BushSlicer::HOME` changes we only need to change in the `I obtain test data file` step.
2. less typing in the actual

Reply:
@miyadav plese use the `I obtain test data file...` approach...thx!

Reply:
got it added this